test/extended/dr/force_redeploy.go

```diff
@@ -0,0 +1,183 @@
+package dr
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	"github.com/pborman/uuid"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	operatorv1client "github.com/openshift/client-go/operator/clientset/versioned/typed/operator/v1"
+)
+
+// forceOperandRedeployment enables forcible redeployment of etcd,
+// kube-apiserver, kube-controller-manager, kube-scheduler
+// operands. This is a necessary part of restoring a cluster from
+// backup.
+
```

Comment:
minor: I think this was meant to be dropped as you have it with the func below

Reply:
Intentional - this is the intent of the file.

Reply:
Updated to reflect intention of documenting file-level purpose.